llvm-tools/src/main/scala/scala/scalanative/llvm/LLVM.scala

```diff
@@ -0,0 +1,158 @@
+package scala.scalanative
+package llvm
+
+import java.io.File
+
+import scala.sys.process.{Process, ProcessLogger}
+import scala.util.Try
+
+import System.{lineSeparator => nl}
+
+object LLVM {
+
+  private def abs(file: File): String =
+    file.getAbsolutePath
+
+  private def running(command: Seq[String]): String =
+    "running" + nl + command.mkString(nl + "\t")
+
+  private def getFiles(base: File, filter: File => Boolean): Seq[File] =
+    (if (filter(base)) Seq(base) else Seq()) ++
+      (Option(base.listFiles()) getOrElse Array.empty flatMap (getFiles(
+        _,
+        filter)))
+
+  private[scalanative] def discover(
```

Comment:
I would prefer to have the object `private[scalanative]`, but we use `compileCSources` as part of scale-native's build to verify that `nativelib` compiles before publishing. Should I put the object as `private[scalanative]` and add a forwarder for `compileCSources` in `ScalaNativePluginInternal`?